src/lib/Model/EventHandlers.ts

```diff
@@ -23,7 +23,8 @@ export class EventHandlers {
         }
         return state;
     });
-    windowResizeHandler = (): void => this.updateState(recalcVisibleRange);
+    windowResizeColumnHandler = (): void => this.updateState(recalcVisibleRange);
+    windowResizeRowHandler = (): void => this.updateState(recalcVisibleRange);
```

Comment:
`windowResizeHandler` is used to recalculate the visible range of elements in the grid and is not directly related to row resize

Reply:
Thanks a lot for your help